akka-slf4j/src/test/resources/logback-test.xml

```diff
@@ -7,7 +7,7 @@
   </appender>
   <appender name="TEST" class="akka.event.slf4j.Slf4jLoggerSpec$TestAppender">
     <encoder>
-      <pattern>%date{ISO8601} level=[%level] marker=[%marker] logger=[%logger] akkaSource=[%X{akkaSource}] akkaAddress=[%X{akkaAddress}] sourceActorSystem=[%X{sourceActorSystem}] sourceThread=[%X{sourceThread}] mdc=[ticket-#%X{ticketNumber}: %X{ticketDesc}] - msg=[%msg]%n----%n</pattern>
+      <pattern>%date{ISO8601} level=[%level] marker=[%marker] logger=[%logger] akkaSource=[%X{akkaSource}] akkaAddress=[%X{akkaAddress}] sourceActorSystem=[%X{sourceActorSystem}] sourceThread=[%X{sourceThread}] mdc=[ticket-#%X{ticketNumber}: %X{ticketDesc} p1: %X{p1} p2: %X{p2}] - msg=[%msg]%n----%n</pattern>
```

Comment:
Does this mean users who use the 'standard' CONSOLE logger without changing the pattern will not see the markers, and only users who use the logstash encoder see them?

Reply:
So this is why we're adding the relevant information both to the message *and* as markers?

Makes some sense I guess, but it's a bit unfortunate: for logstash users it would be much nicer to have the 'variable' information only in the marker, not in the message.

Perhaps as a next step we could move the printing of the markers into the message inside akka-slf4j, and provide an option to omit it for logstash users? Not for this PR though I guess.

Reply:
> Does this mean users who use the 'standard' CONSOLE logger without changing the pattern will not see the markers, and only users who use the logstash encoder see them?

Yes, that's how MDC values work, and as far as I know there is no built-in pattern for printing all MDC values. 

I guess there can be different opinions, but as I see it MDC and marker is for searching and correlating. In the end it's nice to have a human readable log message, even if it may contain some content that are also in the properties.

> Perhaps as a next step we could move the printing of the markers into the message inside akka-slf4j

I'm pretty sure that can be done by expanding the `PatternLayout` to print all MDC values. Fortunately we don't have to solve all logging problems for users since the logging backends are flexible to be extended by others.

Reply:
> Yes, that's how MDC values work, and as far as I know there is no built-in pattern for printing all MDC values.

You should be able to print all MDC values by using `%mdc` or `%X` without a key.

http://logback.qos.ch/manual/layouts.html#mdc

> If no key is given, then the entire content of the MDC will be output in the format "key1=val1, key2=val2".

Reply:
That's excellent, I missed that completely.
I'll try it and add to our docs.

Reply:
I'll include this as a separate PR